src/beatkeeper.rs

```diff
@@ -121,6 +122,7 @@ impl Rekordbox {
         let bar_displays = Value::pointers_to_vals(h, base, &offsets.bar_display[0..decks])?;
         let sample_positions = Value::pointers_to_vals(h, base, &offsets.sample_position[0..decks])?;
         let track_infos = PointerChainValue::pointers_to_vals(h, base, &offsets.track_info[0..decks]);
+        let anlz_files = PointerChainValue::pointers_to_vals(h, base, &offsets.anlz_file[0..decks]);
```

Comment:
As mentioned, `PointerChainValue` re-reads the pointer chain every time, which is more expensive but required for heap values.